tasks/populate-release-notes-images/tests/test-populate-release-notes-images-multiple-images.yaml

```diff
@@ -106,26 +108,28 @@ spec:
           - name: check-result
             image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
             script: |
-              #!/usr/bin/env sh
+              #!/usr/bin/env bash
               set -eux
 
               echo Checking image1arch1...
               image1arch1=$(jq '.releaseNotes.content.images[0]' "$(workspaces.data.path)/data.json")
               test $(jq -r '.architecture' <<< $image1arch1) == "amd64"
               test $(jq -r '.containerImage' <<< $image1arch1) == "registry.redhat.io/product/repo@sha256:abcdefg"
               test $(jq -r '.purl' <<< $image1arch1) == \
-                "pkg:oci/repo@sha256%3Aabcdefg?arch=amd64&repository_url=registry.redhat.io/product"
+                "pkg:oci/repo@sha256%3Aabcdefg?arch=amd64&repository_url=registry.redhat.io/product&tag=9.4.0-1723436855"
               test $(jq -r '.repository' <<< $image1arch1) == "registry.redhat.io/product/repo"
-              test $(jq -rc '.tags' <<< $image1arch1) == '["foo","bar"]'
+              # shellcheck disable=SC2086
+              test "$(jq -rc '.tags' <<< $image1arch1)" == '["9.4-1723436855","9.4.0-1723436855","foo","bar"]'
 
               echo Checking image1arch2...
               image1arch2=$(jq '.releaseNotes.content.images[1]' "$(workspaces.data.path)/data.json")
               test $(jq -r '.architecture' <<< $image1arch2) == "s390x"
               test $(jq -r '.containerImage' <<< $image1arch2) == "registry.redhat.io/product/repo@sha256:deadbeef"
               test $(jq -r '.purl' <<< $image1arch2) == \
-                "pkg:oci/repo@sha256%3Adeadbeef?arch=s390x&repository_url=registry.redhat.io/product"
+                "pkg:oci/repo@sha256%3Adeadbeef?arch=s390x&repository_url=registry.redhat.io/product&tag=9.4.0-1723436855"
               test $(jq -r '.repository' <<< $image1arch2) == "registry.redhat.io/product/repo"
-              test $(jq -rc '.tags' <<< $image1arch2) == '["foo","bar"]'
+              # shellcheck disable=SC2086
+              test "$(jq -rc '.tags' <<< $image1arch2)" == '["9.4-1723436855","9.4.0-1723436855","foo","bar"]'
```

Comment:
The issue here is that `$image1arch2` should be in `""`. So should be:
```
test "$(jq -rc '.tags' <<< "$image1arch2")" == '["9.4-1723436855","9.4.0-1723436855","foo","bar"]'
```

Reply:
Interesting, I thought this would just unquote the variable `$image1arch2`, but it did seem to be okay when tested in a script. Updated.

Reply:
Yeah, it is a bit counter-intuitive. But it does work :)